common/segments/root.xml

```diff
@@ -69,7 +69,7 @@ For terms of use, see http://www.unicode.org/copyright.html
 				<variable id="$CB">\p{Line_Break=Contingent_Break}</variable>
 				<variable id="$CL">\p{Line_Break=Close_Punctuation}</variable>
 				<variable id="$CP">\p{Line_Break=CP}</variable>
-				<variable id="$CM1">\p{Line_Break=Combining_Mark}</variable>
+				<variable id="$CM1">[\p{Line_Break=Combining_Mark}[🏻-🏿]]</variable>
```

Comment:
ICU line.txt does not have this, where is this coming from?

Reply:
I aligned the CLDR rules against the changes for v13.0. In retrospect, that is probably a mistake. I'll hold off on the segmentation changes til v13.0 comes out.

Reply:
Right, Unicode v13.0 rules should go into CLDR 36.1, not 36.